crates/glaredb/src/commands.rs

```diff
@@ -281,7 +281,22 @@ impl RunCommand for MetastoreArgs {
 }
 
 fn build_runtime(thread_label: &'static str) -> Result<Runtime> {
-    let runtime = Builder::new_multi_thread()
+    let mut builder = Builder::new_multi_thread();
+
+    // Bump the stack from the default 2MB.
+    //
+    // We reach the limit when planning a query
+    // with nested views.
+    //
+    // Note that Sean observed the stack size only reaching ~300KB when
+    // running in release mode, and so we don't need to bump this
+    // everywhere. However there's definitely improvements to stack
+    // usage that we can make.
+    // see <https://github.com/GlareDB/glaredb/issues/2390>
+    #[cfg(not(release))]
+    builder.thread_stack_size(4 * 1024 * 1024);
+
+    let runtime = builder
```

Comment:
the sql queries from some of the connectors was causing stack overflows in dev due to referencing many views.